libs/hcp-terraform-operator/config.jsonnet

```diff
@@ -0,0 +1,27 @@
+// libs/hcp-terraform-operator/config.jsonnet
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = [
+  '2.7.1',
+  '2.7.0',
+  '2.6.0',
+  '2.4.1',
```

Comment:
Any reason why we'd generate for all these versions? I'd suppose we can limit to the latest patch of each minor and perhaps only the last three(?) minors. wdyt?

Reply:
Just 'cause we're still only on 2.4.1 at the moment :blush:. I looked through the history of the CRDs in their repo since 2.4.1 and those were the exact tags where the CRDs changed; not sure if that's best practice here? Obviously we'd like to just upgrade the operator at some point but we don't have the time right this moment!

Reply:
there shouldn't be any significant breaking changes between those versions, would use the latest one here

Reply:
Ok, I've removed all but the latest version.